src/Logger/BadgeLogger.php

```diff
@@ -95,25 +95,23 @@ public function log(): void
         try {
             $apiKey = $this->strykerApiKeyResolver->resolve(getenv());
         } catch (CouldNotResolveStrykerApiKey $exception) {
-            $this->showInfo($exception->getMessage());
+            $this->logError($exception->getMessage());
 
             return;
         }
 
-        /*
-         * All clear!
-         */
+        // All clear!
         $this->output->writeln('Sending dashboard report...');
 
-        $this->badgeApiClient->sendReport(
+        $this->strykerDashboardClient->sendReport(
             $apiKey,
             'github.com/' . $buildContext->repositorySlug(),
             $buildContext->branch(),
             $this->metricsCalculator->getMutationScoreIndicator()
         );
     }
 
-    private function showInfo(string $message): void
+    private function logError(string $message): void
```

Comment:
The downstream corresponding log level for `LoggerInterface` is `warning`. But even so `logWarning` would be more accurate. Maybe just gonna go with `logMessage()` which removes confusion. I prefer "log" over "show" here because show is as if you are showing the message: we're not necessarily doing so

Reply:
`logMessage()` is 👌